internal/provider/provider.go

```diff
@@ -132,6 +132,7 @@ func Provider() *schema.Provider {
 			"packetfabric_marketplace_service_requests":  dataSourceVcRequests(),
 			"packetfabric_virtual_circuits":              datasourceBackboneServices(),
 			"packetfabric_point_to_point":                datasourcePointToPoint(),
+			"packetfabric_locations_market":              dataSourceLocationsMarkets(),
```

Comment:
Should it be named `packetfabric_locations_market` or `packetfabric_locations_markets`?